addons/website/static/src/js/tours/survey_tour.js

```diff
@@ -0,0 +1,36 @@
+odoo.define("website.survey.tour", function (require) {
+"use strict";
+
+const wTourUtils = require("website.tour_utils");
+const core = require("web.core");
+const _t = core._t;
+const snippets = require('tour_survey.snippets');
```

Comment:
- You cannot require something which is only defined in themes -> at worst it should require something existing which is **overridden** by themes.

- In fact, is this necessary to override by theme at all? Can't we make a generic tour which says "change the title of the first snippet" generically, like this: `#wrap > section:first-child h1` for example?

Reply:
I don't know if we can do a generic tour for all themes. Your example make the assumption that all the first child have an h1 title but this is not the case...

Reply:
Let see with @tsm-odoo at the meeting in a few minutes